frontend/src/components/account/SettingsPage.tsx

```diff
@@ -453,7 +453,6 @@ export default function SettingsPage({ settings, setSettings, token, setMessage
                 checked={emailPreferences[key]}
                 onChange={() => handlePreferenceChange(event, key)}
                 name={key}
-                color="primary"
```

Comment:
Why did you remove color from this button? I see that in some files you added `color` parameter to the button. What is different here?

Reply:
From the PR description: "With mui 5 the breakpoints have changed and for the same behavior each breakpoint needs to be upgraded by one (e.g. "xs" -> "sm"). I have made this change by running a codemod provided by mui."